### DISCUSSION TITLE
'display output is not active' on macOS  for ubuntu 22.04 arm64

### DISCUSSION THREAD

**Discussion by FBaxon** [utmapp/UTM#6503] [UTM Help (macOS)]
apple m1, sonoma 14.5
the latest UTM from the official website

Actually, it still worked yesterday, running normally after waiting for a few seconds when meeting the trouble 'display output is not active'. So I didn't minded it. But today it just keeps the state 'display output is not active'

**Comment by macuserguru** [comment]
I see frome some VMs 'display output is not active' at starting but disappears after a short time.

**Comment by macuserguru** [comment]
M1 MacMini, Sonoma, UTM (from appstore) latest versions.
Fedora 40 (GNOME, Budgie) - this show for a while, till login coming.
![Fedora 40 (GNOME, Budgie) 2024-07-20 um 14 34 49](https://github.com/user-attachments/assets/48eea2b8-548c-4314-8282-603d27c87761)

**Comment by FBaxon** [comment]
I finally made it by reinstalling it, and the key was to choose 'GPU acceleration supported' at the beginning

**Reply by iamsoum** [reply]
Reinstalling what? UTM? or the VM inside UTM? And can't we not edit the VM settings in UTM to enable GPU acceleration?

**Reply by raweber42** [reply]
Enabling the GPU acceleration (can be selected when selecting `Linux` and not `other` for OS) fixed it for me, thanks!

**Reply by FBaxon** [reply]
reinstall the VM

**Comment by MinhHieu2112** [comment]
https://youtu.be/11ZMdcJ23sk?si=u-iuRh94UNUdNJat
I think this video will help you a lot, I encountered some errors like "display output is not active", "uefi interactive shell v2 2" and stuck on window. But now, everything is good and I can go on working with some window's apps on my Mac.

**Comment by tawek** [comment]
Connect serial device and it might help to debug your setup. For me it was volume password (that is commonly offered in server setups). VM was starting and asking for a LUKS password (silly me).
